spotbugs/etc/messages.xml

```diff
@@ -93,73 +93,149 @@ This plugin contains all of the standard SpotBugs detectors.
   <BugCategory category="CORRECTNESS">
     <Description>Correctness</Description>
     <Abbreviation>C</Abbreviation>
-    <Details>Probable bug - an apparent coding mistake
+    <Details>
+<![CDATA[
+<p>
+    Probable bug - an apparent coding mistake
             resulting in code that was probably not what the
-            developer intended. We strive for a low false positive rate.</Details>
+            developer intended.
+</p>
+<p>
+             We strive for a low false positive rate.
+</p>
+]]>
```

Comment:
For now we can NOT use `<![CDATA[` in `<Details>` in `<BugCategory>`, according to Javadoc in `BugCategory.java`:

```java
    /**
     * ...
     * @param detailText
     *            full description of the bug category (no HTML markup, may be
     *            null)
     */
    public BugCategory(String category, String shortDescription, String abbrev, String detailText) {
```

So please revert changes for BugCategory, it should keep using plain text instead of HTML markup in `<![CDATA[`.